cupy/cuda/nccl.py

```diff
@@ -0,0 +1,14 @@
+"""
+NCCL Wrapper
+
+Use `cupy_backends.cuda.libs.nccl` directly in CuPy codebase.
```

Comment:
`cupyx/_runtime.py` uses `cupy.cuda.nccl`, so it should be updated.

Reply:
Good catch, thanks! Fixed in 437ee18.